src/index.html

```diff
@@ -1,13 +1,413 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>MyBike</title>
+    <link
+      rel="shortcut icon"
+      href="images/MyBiKE.svg"
+      type="image/x-icon"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@500;700&family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="header page__section">
+      <div class="container">
+        <div class="header__content">
+          <div class="header__top">
+            <a href="#home" class="logo">
+              <img 
+                 src="images/MyBiKE.svg" 
+                 alt="MyBike"
+                 class="logo__image"
+              >
+            </a>
+            <div class="header__links">
+              <a 
+                href="+1 234 555-55-55" 
+                class="icon icon--phone header__call"
+              ></a>
+              <a 
+                href="#menu" 
+                class="icon icon--menu_opener"
+              ></a>
+            </div>
+          </div>
+
+          <h1 class="header__title">
+            Take the Streets
+          </h1>
+        </div>
+      </div>
+    </header>
+
+    <nav class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="menu__content">
+          <div class="menu__top">
+            <a href="#home" class="logo">
+              <img 
+                src="images/MyBiKE.svg" 
+                alt="MyBike"
+                class="logo__image"
+              >
+            </a>
+
+            <a href="#" class="icon icon--cross"></a>
+          </div>
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a href="#home" class="menu__link">Home</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#about" class="menu__link">About Us</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#compare-bikes" class="menu__link">Compare bikes</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#details" class="menu__link">Details</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#contacts" class="menu__link">Contacts</a>
+            </li>
+          </ul>
+
+          <a 
+            href="tel.:+1 234 5555-55-55" 
```

Comment:
```suggestion
            href="tel:+123455555555" 
```
Спробуй так можливо зникне 404